tests/Feature/BookingRequestControllerTest.php

```diff
@@ -112,7 +112,7 @@ public function user_can_create_booking_request()
         $this->createReservationAvailabilities($date, $room);
 
         $start = Carbon::parse($date);
-        $end = $start->copy()->addMinutes(4);
+        $end = $start->copy()->addMinutes(34);
 
```

Comment:
could you add 60 instead of 34 :P ? Bookings are suppose to end with :15, :30, :45, :00 ! That feature is coming or theres currrently a bug ^^

Reply:
done!